package.json

```diff
@@ -43,6 +43,7 @@
     "@octokit/rest": "^16.43.1",
     "@sentry/electron": "^2.5.3",
     "@types/getos": "^3.0.1",
+    "bind-decorator": "^1.0.11",
```

Comment:
Instead of adding a new dependency can you make a custom Autobind decorator. Learn about them [here](https://www.typescriptlang.org/docs/handbook/decorators.html)